packages/core/data-transfer/src/file/providers/source/index.ts

```diff
@@ -121,20 +138,21 @@ class LocalFileSourceProvider implements ISourceProvider {
       [
         inStream,
         new tar.Parse({
+          // find only files in the assets/uploads folder
           filter(filePath, entry) {
             if (entry.type !== 'File') {
               return false;
             }
-
-            const parts = filePath.split('/');
-            return parts[0] === 'assets' && parts[1] === 'uploads';
+            return isFilePathInDirname('assets/uploads', filePath);
```

Comment:
Shouldn't we rely on path.posix.join(assets, uploads) instead of a regular string? I know it won't make a real difference here, it's more about consistency

Reply:
I think this is fine, the function requires a posix path and we are providing one. And we use paths like that in many places throughout the codebase. I'm more trusting of this string that I know exactly what it is rather than trusting posix.join not to change it to something like './assets/uploads' or 'assets/uploads/' without me knowing for sure (even though those cases would be fine as seen in the tests).